test/TrialsTests.jl

```diff
@@ -233,4 +233,17 @@ else
 
 end
 
+trial = BenchmarkTools.Trial(BenchmarkTools.Parameters(), [1.0, 1.01], [0.0, 0.0], 0, 0)
+@test sprint(show, "text/plain", trial) == """
+BenchmarkTools.Trial: 2 samples with 1 evaluation.
+ Range (min … max):  1.000 ns … 1.010 ns  ┊ GC (min … max): 0.00% … 0.00%
+ Time  (median):     1.005 ns             ┊ GC (median):    0.00%
+ Time  (mean ± σ):   1.005 ns ± 0.007 ns  ┊ GC (mean ± σ):  0.00% ± 0.00%
+
+  █                                                       █  
```

Comment:
@tecosaur one of the bars here is misaligned?

Reply:
It looks fine to me.

![image](https://user-images.githubusercontent.com/20903656/129333441-4d3a2c41-12d6-4bf1-b6a0-8a1a219efb38.png)